browser/index.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-env browser */
+const snoowrap = require('..');
+const oauthInfo = require('../oauth_info.json');
+
+const r = new snoowrap({
```

Comment:
Right! I haven't think about that! This would be a huge security issue. Your suggestion sounds great, I'll patch it once I finish working on the new inline media feature.  
Also probably I'll replace `live-server` with `five-server` (an active fork) since it's unmaintained and depends on outdated libraries that have security vulnerabilities.  
I think `oauth_info.json` should be restructured, you can't run both snoowrap instances (the one authenticated with username/password & the one authenticated with a refresh token) at the same time since they require different client ids.

Reply:
Update: we're using Vite now!